src/platform/packages/shared/kbn-alerting-types/rule_types.ts

```diff
@@ -15,6 +15,7 @@ import type {
 import type { Filter } from '@kbn/es-query';
 import type { RuleNotifyWhenType, RRuleParams } from '.';
 
+export type RuleTypeSolution = 'observability' | 'security' | 'stack';
```

Comment:
Had to move this here to avoid circular dependencies between plugins and packages